Task/scripts/run-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+docker network create grid	# create grid
+docker run -d -p 4444:4444 --net grid --name selenium-hub selenium/hub:3.141.59-radium	# run hub
+docker run -d --net grid -e HUB_HOST=selenium-hub -v /dev/shm:/dev/shm selenium/node-chrome:3.141.59-radium	# add chrome node
+docker run -d --net grid -e HUB_HOST=selenium-hub -v /dev/shm:/dev/shm selenium/node-firefox:3.141.59-radium	# add firefox node
+cd ~	# go to main directory
+cd GitRepos/ssFirstTask/Task	# go to project directory
+mvn clean test	# run maven tests
+docker kill $(docker ps -q)	# kill all containers
```

Comment:
This will kill all running containers, could we be more specific and turn down only once we launched?

Reply:
Yes, we can kill each container one by one, but I think in this case killing all containers is more useful because  we any way use only one container.